docs/_includes/counter.adoc

```diff
@@ -1,9 +1,66 @@
 ////
 Included in:
 
-- user-manual: Counter Attributes
+- user-manual: Counters
 ////
 
+// See issue #550, #561, https://github.com/asciidoctor/asciidoctor/issues/1559
+// Focus on simple use case.
+// {counter:name:start_value} is only for first use.
+// document attributes and counters are NOT the same thing, but modifying a document attribute with the same name as the counter modifies the counter at the same time.
+
+Counters are used to generate sequences of numbers or latin characters.
+Possible uses are to create inline itemised lists, paragraph numbers, or serial numbers.
+
+Counters are created and used by prefixing the counter name with `counter:` and surrounding it with curly brackets.
+(Counter names follow the same rules as attribute names).
+The default start value for a counter is 1, so if you want a sequence starting at 1 you can simply use:
+
+ {counter:i}. Apples, {counter:i}. Oranges, {counter:i}. Pears.
+ 
+Which gives:
+
+{counter:i}. Apples, {counter:i}. Oranges, {counter:i}. Pears.
+
+The counter is incremented and displayed every time `counter:` is used.
+To skip an item in the sequence (i.e. increment the counter without displaying it), use `counter2` instead:
+
+ {counter2:item_number}
+
+To display the current value of the counter without incrementing it, reference it:
+
+ {counter:para_number} This paragraph is number {para_number}.
+
+NOTE: "increment" means add 1 if the current value is an integer, or choose the next character of the latin alphabet if it is anything else.
```

Comment:
I'd prefer this as:

"increment" means advance to the next value in the sequence.
If the counter value is an integer, add 1.
If the counter value is a character, move to the next character in the Latin alphabet.